paasta_tools/utils.py

```diff
@@ -2686,6 +2687,9 @@ def get_cluster_aliases(self) -> Dict[str, str]:
     def get_hacheck_match_initial_delay(self) -> bool:
         return self.config_dict.get("hacheck_match_initial_delay", False)
 
+    def get_paasta_api_number_workers(self) -> int:
+        return self.config_dict.get("paasta_api_number_workers", 4)
+
```

Comment:
Does this get called anywhere?

Reply:
whoops - didn't save this before before commiting